imports/startup/server/accounts.js

```diff
@@ -76,12 +100,30 @@ if (Meteor.users.find().fetch().length === 0) {
     );
 
     Roles.addUsersToRoles(id, userData.roles);
+
   });
+
+  //Find Admin User ID.
+  var adminUser = Accounts.findUserByEmail("admin@example.com");
+
+  //Creating New 8 Digit Alphanumeric Password.
+  var randomPassword = passwordGen(8);
+  var newPassword = randomPassword.substring(1);
+
+  //Applying Password to Meteor Admin User and Logging to Server.
+  if(Meteor.settings.adminPassword==="password"){
+    //ONLY IF PASSWORD HAS NOT BEEN SET ON ADMIN,
+    //Apply Password to Meteor Admin User and Logging to Server 
+    Accounts.setPassword(adminUser,newPassword);
+    console.log("Admin Email: " + "admin@example.com");
+    console.log("Admin Password: " + newPassword);
+  }
```

Comment:
This works perfectly now. Congratulations on that @Max2020q 

But we can make it even better, add an else part to this if condition.
Inside else, Set the password of the admin user to the `Meteor.settings.adminPassword`. This will update the password when we change the password in the settings.json file.

Reply:
That is a great suggestion. I have tried to implement this and add comments to illustrate this solution.

Reply:
Hey @Max2020q, the solution for changing password looks good.

But since it's inside the else if condition `else if(Meteor.users.find().fetch().length === 0) {..}`, it runs only once for the very first time when we don't have an admin user in the database. 

Can you move all the password change logic outside this else if condition, so it runs every time we run the meteor server?

Reply:
@Gauravjeetsingh  Thank you for the review. In this latest commit I moved the logic of the password change outside the "if else" statement from the review above.

https://github.com/BRL-CAD/OGV-meteor/pull/91/commits/8a2748cbf7f5b472f6d472a175daa97760d34437